core/integration/module_python_test.go

```diff
@@ -269,13 +269,14 @@ build-backend = "poetry.core.masonry.api"
 		},
 	}
 
-	for _, tc := range testCases {
+	for i, tc := range testCases {
 		tc := tc
 
 		t.Run(fmt.Sprintf("%s/%s", tc.name, tc.path), func(ctx context.Context, t *testctx.T) {
 			c := connect(ctx, t)
 
 			out, err := daggerCliBase(t, c).
+				WithWorkdir(fmt.Sprintf("/work/%s-%d", tc.name, i)).
```

Comment:
Hmm, these definitely shouldn't be conflicting right? It's the weird cache behavior thing (again?).

Can we add a quick note about this inline, just so it doesn't accidentally get removed? (and we can remove it if we think we have a longer term fix).

Reply:
Yeah, sure. This fix is for uv's cache however, not ours.